compact/definitions.xml

```diff
@@ -185,15 +185,17 @@ The unused IDs below are saved for future use.
       - Barrel   subassembly ID: 111
       - EndcapP  subassembly ID: 112
       - EndcapN  subassembly ID: 113
+      - oHCal    subsystem  ID: 116
 
-      - Unused IDs: 116-119
+      - Unused IDs: 117-119
     </documentation>
     <constant name="HCalSubAssembly_ID" value="110"/>
     <constant name="HCalBarrel_ID"      value="111"/>
     <constant name="HCalEndcapP_ID"     value="112"/>
     <constant name="HCalEndcapN_ID"     value="113"/>
     <constant name="PassiveSteelRingEndcapP_ID" value="114"/>
     <constant name="HCalEndcapPInsert_ID" value="115"/>
+    <constant name="oHCalBarrel_ID"      value="116"/>
```

Comment:
I was concerned about downstream effects of re-using the HCALBarrel_ID - I assume this keys into the reconstruction at some point and teh structure of the real HCAL is quite different from the ATHENA HCAL.  I thought it might be better to separate them to allow the possibility to have both for a limited time, in case this crashes and burns at the reconstruction stage and needs a lot more work to be useful. 

Happy to just change it to HCALBarrel_ID if it won't cause dramatic downstream disruption.

Reply:
Change made and committed